### PR TITLE
Add category page

### PR DESCRIPTION
**Pull request by anupkgurung**
Added page for category with Quiz category

### CODE REVIEW

File: src/page/Category/Category.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { LinkButton, QuizCard } from "../../component";
+
+const getCategories = async (setCategories) => {
+    const { data: { quizzes } } = await axios.get("api/category/quiz");
+    setCategories(quizzes)
```
Comment (maitrakhatri) [inline src/page/Category/Category.jsx]: suggestion: do API calls in try block and use catch for error handling

**Review by maitrakhatri — APPROVED** [review]
Clean code and good folder structure

File: src/page/Home/Home.jsx [hunk 2]
```diff
@@ -1,7 +1,32 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { QuizCard } from "../../component/QuizCard/QuizCard"
+
+const getCategories = async (setCategories) => {
+    try{
+        const { data : {categories} } = await axios.get("api/category/");
+        setCategories(categories)
+    }catch({response}){
+        console.log(response)
```
Comment (ranitasahaa1230) [inline src/page/Home/Home.jsx:10]: Remove this console.log if not needed

**Review by ranitasahaa1230 — APPROVED** [review]
Code can be understood easily.
Good use of semantic tags
Do add a screenshot or a GIF in the description

**Pull request merged**